engine/install/binaries.md

```diff
@@ -9,23 +9,37 @@ redirect_from:
 - /installation/binaries/
 ---
 
-> **Note**: You may have been redirected to this page because there is no longer
-> a dynamically-linked Docker package for your Linux distribution.
+> **Important**
+>
+> This page contains information on how to install Docker using binaries. These
+instructions are mostly suitable for testing purposes. We do not recommend
+installing Docker using binaries in production environments as they will not be
+updated automatically with security updates. The Linux binaries described on this
+page are statically linked, which means that vulnerabilities in build-time
+dependencies are not automatically patched by security updates of your Linux
+distribution.
+>
+> Updating binaries is also slightly more involved when compared to Docker packages installed using a package manager or through Docker Desktop, as it requires (manually) updating the installed version whenever there is a new release of Docker.
+>
+> Also, static binaries may not include all functionalities provided by the dynamic packages.
+>
+> On Windows and Mac, we recommend that you install [Docker Desktop](../../desktop/index.md) instead. For Linux, we recommend that you follow the instructions specific for your distribution.
+{: .important}
```

Comment:
Perhaps wrap this for all lines, and put a `>` on each line (makes it easier to see where the "quote" starts and ends when reading the Markdown source).

```suggestion
> **Important**
>
> This page contains information on how to install Docker using binaries. These
> instructions are mostly suitable for testing purposes. We do not recommend
> installing Docker using binaries in production environments as they will not be
> updated automatically with security updates. The Linux binaries described on this
> page are statically linked, which means that vulnerabilities in build-time
> dependencies are not automatically patched by security updates of your Linux
> distribution.
>
> Updating binaries is also slightly more involved when compared to Docker packages
> installed using a package manager or through Docker Desktop, as it requires
> (manually) updating the installed version whenever there is a new release of
> Docker.
>
> Also, static binaries may not include all functionalities provided by the dynamic
> packages.
>
> On Windows and Mac, we recommend that you install [Docker Desktop](../../desktop/index.md)
> instead. For Linux, we recommend that you follow the instructions specific for
> your distribution.
{: .important}
```